specification.md

```diff
@@ -145,10 +163,47 @@ configuration.
 See sections under the [Operation](#operation) section for the details of the
 message sequences.
 
+The WebSocket transport is typically used when it is necessary to have instant
+communication ability from the Server to the Agent without waiting for the Agent
+to poll the Server like it is done when using the HTTP transport (see below).
+
+## Plain HTTP Transport
+
+The second supported transport for OpAMP protocol is plain HTTP connection. The
+Agent is an HTTP client and the Server is an HTTP client server. The Agent makes
+POST requests to the Server. The body of the POST request and response is a
+[binary serialized Protobuf](https://developers.google.com/protocol-buffers/docs/encoding)
```

Comment:
I think it would be worth considering allowing JSON serialized Protobuf as well. It typically has less dependencies and in some cases (e.g. mobile apps, web browsers, Node.js) is preferred over Protobuf

Reply:
Yes, that's fairly easily achievable via Content-Type header and also easy to implement on the server-side to support both JSON and binary Protobufs.

Reply:
Added as Open Question.